tests/common/util.rs

```diff
@@ -69,39 +70,105 @@ pub struct CmdResult {
     /// zero-exit from running the Command?
     /// see [`success`]
     pub success: bool,
-    /// captured utf-8 standard output after running the Command
+    /// captured standard output after running the Command
     pub stdout: String,
-    /// captured utf-8 standard error after running the Command
+    /// captured standard error after running the Command
     pub stderr: String,
```

Comment:
Shouldn't this be stored as `Vec<u8>`?  Same for `stdout`.

Reply:
> Shouldn't this be stored as `Vec<u8>`? Same for `stdout`.

That's the big problem. A lot of tests depend on the fields of `CmdResult`.
Changing this right now would break the whole build. So, as mentioned in the original PR comment,
for now, we just add a bunch of public methods.
After we have refactored the tests to only use these functions, we can make the fields private and change the type.

The refactor shouldn't take more than 1-2 days.